zcash_client_backend/src/data_api.rs

```diff
@@ -159,21 +165,30 @@ pub trait WalletRead {
     /// with which they are associated.
     fn get_nullifiers(&self) -> Result<Vec<(AccountId, Nullifier)>, Self::Error>;
 
-    /// Return all spendable notes.
-    fn get_spendable_notes(
+    fn get_all_nullifiers(&self) -> Result<Vec<(AccountId, Nullifier)>, Self::Error>;
+
+    /// Return all unspent notes.
+    fn get_unspent_sapling_notes(
         &self,
         account: AccountId,
         anchor_height: BlockHeight,
     ) -> Result<Vec<SpendableNote>, Self::Error>;
 
-    /// Returns a list of spendable notes sufficient to cover the specified
+    /// Returns a list of unspent notes sufficient to cover the specified
     /// target value, if possible.
-    fn select_spendable_notes(
+    fn select_unspent_sapling_notes(
         &self,
         account: AccountId,
         target_value: Amount,
         anchor_height: BlockHeight,
     ) -> Result<Vec<SpendableNote>, Self::Error>;
+
+    #[cfg(feature = "transparent-inputs")]
+    fn get_unspent_transparent_utxos(
+        &self,
+        address: &TransparentAddress,
+        anchor_height: BlockHeight,
```

Comment:
This is not actually an anchor height. But it's okay as long as the documentation explains.